Chapters/1-5-Chapter5.md

```diff
@@ -0,0 +1,155 @@
+## Chapter 5
+
+The city was carved from the body of an old mountain, curled upward as if the land itself was an ocean wave. Massive veins of white marble meandered between gray, banded rock, and striped through the roads and walls like an unmoving spectre casting shadows through time. Every building faced proudly toward Ara, carefully reflecting every glint of orange and red and infrared. High upon the mountain within the largest vein of marble sat the senate, with its crown, a the gilded halo of the Palinian circle. I imagined being one of the people that carved out their piece of the mountain to call home. _It must have taken ages_.
+
+"Been a while, has it?" Fajro said once we got off.
+
+"Yes. At least a few ma'tova."
+
+"The beauty of this place never changes. In Holrag, we call this place "The diamond of Palinan."
+
+"Even after years of coming here, it hasn't lost its luster?"
+
+"Every time, it seems like a distant memory. This city changes you the minute you join the throng. Every day here is another lifetime."
+
+"Let's make this one count."
+
+The base of the city was brightly lit, but there were large spots of shade behind giant curved mirrors in the distance. They were continuously focusing and harvesting Ara's light onto pipes of water to deliver steam to all of Shaki. Near the railyard an air of bustle filled the area with hordes of trainferriers bringing cargo to and from the trains; trade scribes in toe trying to keep up. Their heads would occasionally turn toward a place settled the desert, visible to all the city. A thin tower showed two sets of shadowed vertical lines against the bright light.
+
+_The timekeeper. I remember now._
+
+I helped Fajro walk from the railyard to the easternmost funicular station. It was not terribly distant, but the sluggish pace was wearing thin on the patience of passersby trying to get around us.
+
+"I regret that I am slowing you down," he said, acknowledging his frustration.
+
+"No such talk. These people can afford to slow down a little bit."
```

Comment:
Fajro is talking about Liffa, and Liffa is talking about other people. what?

Reply:
Yea this was confusing. Fixed.